### ISSUE TITLE
Report for the newsletter

### ISSUE THREAD

**Issue by manez** [Islandora-CLAW/CLAW#102]
This is kind of a sideways approach to an issue, but since this is where we're tracking stuff that needs to get done, I figure this is as good a place as any. 

I'll be sending out another Islandora newsletter in December and it would be great to get reports from one or more sprinters about their experience of this event, so we can get the word out to more of the community.

**Comment by ruebot** [comment]
_pings_ @Islandora-Labs/7-x-2-x-sprinters

**Comment by whikloj** [comment]
This was the first sprint right, not that I had much involvement. But from the work done it seemed like a win.

**Comment by bradspry** [comment]
[Documentation] Does not work with VirtualBox 5.0 #43 
https://github.com/Islandora-CLAW/CLAW/issues/43

Gathered a few more results from users.  Hope was to shine a little bit of light onto vagrant build issues.

Most of the Sprint however was dedicated to learning, engaging, and listening, with the goal of gaining some semblance of understanding for the future architecture and proposed engineering directions.   I was able to get a foothold of understanding, enough to comprehend that developers were debating between Drupal 7 and Drupal 8's capabilities.   I was able to conduct enough research to formulate and post an opinion:
https://groups.google.com/d/msg/islandora/YyPL_LQpktA/bB1h-kDvBwAJ
